.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "blocky development",
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu-22.04",
+  "features": {
+    "ghcr.io/devcontainers/features/go:1": {
+      "version": "1.21"
```

Comment:
I would try to avoid the fix golang version here, otherwise we need to maintain the current version twice, here and in go.mod. Imho it is better to use the latest 1.x here

Reply:
Sorry had automerge enabled.
Will open another PR for this.